src/assets/recipeData.json

```diff
@@ -3,7 +3,7 @@
     "material": "Adamantium Filaments",
     "craftable": false,
     "stat": "Health",
-    "rarity": "Common",

```

Comment:
I Prefer to keep it in upper case

Reply:
I'll take it back to upper case. I was just trying to match campaignConfig